tests/test_utils_cache.py

```diff
@@ -227,6 +228,28 @@ def test_cli_scan_cache_verbose(self) -> None:
         )
 
 
+    def test_cli_scan_missing_cache(self) -> None:
+        """Test output from CLI scan cache when cache does not exist.
+
+        End-to-end test just to see if output is in expected format.
+        """
+        args = Mock()
+        args.verbose = 0
+        args.dir = "/tmp/empty"
+
+        with capture_output() as output:
+            ScanCacheCommand(args).run()
+
+        expected_output = f"""
+        Cache directory not found: /tmp/empty
+        """
+
+        self.assertListEqual(
+            output.getvalue().replace("-", "").split(),
+            expected_output.replace("-", "").split(),
+        )
```

Comment:
No need to replace the "-" here (it's just a quite hacky way to test the tables without the "---------...")

```suggestion
        self.assertListEqual(output.getvalue().split(), expected_output.split())
```